nat-lab/tests/test_lana.py

```diff
@@ -105,13 +105,17 @@ def get_moose_db_file(container_tag, container_path, local_path):
 
 async def wait_for_event_dump(container, events_path, nr_events):
     start_time = asyncio.get_event_loop().time()
-
+    events = []
     while asyncio.get_event_loop().time() - start_time < DEFAULT_CHECK_TIMEOUT:
         get_moose_db_file(container, CONTAINER_EVENT_PATH, events_path)
         events = fetch_moose_events(events_path)
         if len(events) == nr_events:
             return events
         await asyncio.sleep(DEFAULT_CHECK_INTERVAL)
+    print(
```

Comment:
Is this print visible anywhere?

Reply:
Yes, printed during pytest when a test fail to aid debugging

Reply:
Oh, I did not know that prints are visible when a test fails